### DISCUSSION TITLE
A1111 prompt weighting and more

### DISCUSSION THREAD

**Discussion by BlenderNeko** [comfyanonymous/ComfyUI#521] [Show and tell]
With the latest update to ComfyUI it is now possible to use the [AdvancedClipEncode](https://github.com/BlenderNeko/ComfyUI_ADV_CLIP_emb) node which gives you control over how you want prompt weights interpreted and normalized. Among other things this gives you the option to interpret the prompt weights the same way A1111 does things (something that seemed to be a popular request). So go grab the node and play around with it!

**Comment by azureprophet** [comment]
Hi there, how do you use this? Do you use the same syntax as A1111 like (a frog made of gasoline:0.9)

**Reply by BlenderNeko** [reply]
install by going to your custom_nodes folder, and calling `git clone https://github.com/BlenderNeko/ComfyUI_ADV_CLIP_emb.git`, the syntax for up/down weighting is the [same](https://blenderneko.github.io/ComfyUI-docs/Interface/Textprompts/#up-and-down-weighting) as in comfy

**Reply by TomLucidor** [reply]
Didn't that repo also have advanced weight adjusters? What do they do?

**Comment by buggsi** [comment]
Also this solution https://github.com/comfyanonymous/ComfyUI/discussions/118#discussioncomment-10377850
